src/components/SlackSignup.tsx

```diff
@@ -4,9 +4,13 @@ import styled from "styled-components";
 
 interface Props {}
 
-const emailRegex = /^[-!#$%&'*+\/0-9=?A-Z^_a-z{|}~](\.?[-!#$%&'*+\/0-9=?A-Z^_a-z`{|}~])*@[a-zA-Z0-9](-*\.?[a-zA-Z0-9])*\.[a-zA-Z](-?[a-zA-Z0-9])+$/;
+const _emailRegex = /^[-!#$%&'*+\/0-9=?A-Z^_a-z{|}~](\.?[-!#$%&'*+\/0-9=?A-Z^_a-z`{|}~])*@[a-zA-Z0-9](-*\.?[a-zA-Z0-9])*\.[a-zA-Z](-?[a-zA-Z0-9])+$/;
```

Comment:
I don't care either way, but what's the purpose of this?  Since it's not exported from the module, the name really only matters locally.

Reply:
It's just a syntax hint about where the function came from.  I picked that up from C# stuffs way back. I'll remove them.

Reply:
For me, it says "this is a private function that exists outside of your class, but in the same file"